assets/src/dashboard/app/reducer/stories.js

```diff
@@ -58,6 +60,32 @@ function storyReducer(state, action) {
         },
       };
 
+    case ACTION_TYPES.TRASH_STORY:
+      return {
+        ...state,
+        storiesOrderById: state.storiesOrderById.filter(
+          (id) => id !== action.payload.id
+        ),
+        totalStories: state.totalStories - 1,
+        stories: Object.keys(state.stories).reduce((memo, storyId) => {
```

Comment:
I think it makes sense to filter it out.  If sometime in the future we decide to access `stories` directly (for whatever reason), it wouldn't be immediately obvious that it might contain "deleted" stories that have yet to be removed.

Reply:
I am big on reducing any stale data here. Because if they try to rename it or it shows up and they do something, the app will error out.

Reply:
Cool sounds good!